value.go

```diff
@@ -53,6 +53,16 @@ func Null(iso *Isolate) *Value {
 	return iso.null
 }
 
+// NewStringFromByteArray returns V8::string from byte array. Converts each byte into equivalent character.
+func NewStringFromByteArray(iso *Isolate, val []byte) (*Value, error) {
```

Comment:
NewStringFromBytes would be a shorter name.  Referring to the argument as a ByteArray doesn't make it any clearer and could even seem to refer to a JS array before looking at the parameter type.

The parameter name could be named bytes to be consistent with what it is referred to as in the function name.

Also, I've opened https://github.com/rogchap/v8go/pull/277 since this should actually be returning a String type.  We should add that type before adding a constructor for a string, so we don't need a breaking change to change it from Value to String later.

```suggestion
func NewStringFromBytes(iso *Isolate, bytes []byte) (*String, error) {
```